diagnostic.md

```diff
@@ -27,7 +27,9 @@ on the body element. Use that variable to increase font-size of `h1`
 elements by 30%.
 
 ```scss
-// your response here
+h1 {
+  font-size: $base-font-size * .3;
```

Comment:
Correct, but the math is wrong. `h1`'s need their font size **increase** by 30% so it would be:
```scss
h1 {
 font-size: $base-font-size * 1.3;
}
```